### PR TITLE
significant individuals filing store

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*https://github.com/bcgov/entity/issues/18677

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-62-goo6ef6x.web.app

File: btr-web/btr-main-app/pages/ownerChange.vue [hunk 1]
```diff
@@ -8,25 +8,23 @@
         <IndividualPersonAddNew />
       </div>
     </div>
-    <IndividualPersonSummaryTable class="mt-5" :individuals="individuals" />
+    <IndividualPersonSummaryTable class="mt-5" :individuals="currentSIFiling.significantIndividuals || []" />
```
Comment (hfekete) [inline btr-web/btr-main-app/pages/ownerChange.vue:11]: Is it just the ones being currently filed (will current filing contain already existing ones ?)

**Review by hfekete — APPROVED** [review]

File: btr-web/btr-main-app/pages/ownerChange.vue (hunk 1 above)
Comment (kialj876) [reply btr-web/btr-main-app/pages/ownerChange.vue:11]: I will mock that out as part of https://github.com/bcgov/entity/issues/18664

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-62-goo6ef6x.web.app

**Pull request merged**